uint/src/uint.rs

```diff
@@ -50,6 +50,26 @@ macro_rules! impl_map_from {
 	}
 }
 
+#[macro_export]
+#[doc(hidden)]
+macro_rules! impl_try_from_for_primitive {
```

Comment:
IMHO, `impl_try_into_primitive` seems like a more accurate and unambiguous name.

Reply:
I don't mind `impl_try_from_for_primitive` tbh, seems explicit and obvious to me?

Reply:
I was trying to make it consistent with other naming here like `impl_mul_for_primitive`, `impl_heapsize_for_uint`. It might be more explicit about which trait implemented for who with this convention?